ContactsApp/Contact.swift

```diff
@@ -2,14 +2,15 @@
 //  Contact.swift
 //  ContactsApp
 //
-//  Created by Davide Mendolia on 30/09/15.
-//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//  Created by Rohit Bansal on 04/10/15.
 //
 
 import Foundation
 
-protocol Contact {
-    var firstName: String { get }
-    var lastName: String { get }
-    var phonenumber: String { get }
+struct Contact : ContactProtocol {
```

Comment:
The idea behind using protocols as based entity, is to keep as paradigm a Protocol Oriented Programming, and Concrete type will get explicit names.
https://developer.apple.com/videos/play/wwdc2015-408/
http://www.raywenderlich.com/109156/introducing-protocol-oriented-programming-in-swift-2